test/IECoreScene/PointsAlgoTest.py

```diff
@@ -522,5 +522,48 @@ def testRaisesExceptionIfTypesAreIncompatible( self ) :
 		self.assertRaises( RuntimeError, lambda : IECoreScene.PointsAlgo.mergePoints( [pointsA, pointsB] ) )
 
 
+class SegmentPointsTest( unittest.TestCase ) :
```

Comment:
Can we get tests for `MeshAlgo::segment()` and `CurvesAlgo::segment()` as well?

Reply:
curves tests: https://github.com/ImageEngine/cortex/pull/689/commits/2a09f4346cb2957b5df1b2a9470e23fdf78990f9
mesh tests: https://github.com/ImageEngine/cortex/pull/689/commits/5e5f7de74b3c17b86348443704abe73a6d4bb23a